lale/lib/lale/join.py

```diff
@@ -107,7 +107,9 @@ def validate_hyperparams(cls, pred=None, **hyperparams):
                         and right_table_name in sub_list_tables
                     ):
                         raise ValueError(
-                            "One of the composite keys tried joining more than two tables!"
+                            "ERROR: Composite key involving '{}' and '{}' tables is bad! Tried joining more than two tables!".format(
```

Comment:
again, can we include the actual failing expression here as well (sub_key._expr).  Also, it is a bit odd to complain about joining more than two tables, but only having two tables in the error message.  It would be better to have (at least) 3.

Reply:
Discussed and will incorporate this later if needed.